pkgs/by-name/me/mescc-tools/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchgit
+, m2libc
+, which
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mescc-tools";
+  version = "unstable-2023-05-23";
+
+  src = fetchgit {
+    url = "https://git.savannah.gnu.org/git/mescc-tools.git";
+    rev = "632357f807b7a5e518d1df368d465aebc7b4d139";
+    hash = "sha256-jFDrmzsjKEQKOKlsch1ceWtzUhoJAJVyHjXGVhjE9/U=";
```

Comment:
```suggestion
    hash = "sha256-vDgafhwxy5dNyiPVawlzbrrDIWsbXvd6x2CqN/ME+fc=";
```

Reply:
For some reason this hash doesn't seem to be stable? `fetchSubmodules` is meant to be false but sometimes it tries anyways? But when I rebuilt the problem went away :/

Reply:
i didn't realise `fetchSubmodules` defaults to `true`. i've changed it to force `false`